packages/datatrak-web-server/src/__tests__/utils/processSurveyResponse.test.ts

```diff
@@ -2,23 +2,18 @@
  * Tupaia
  *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
  */
-import moment from 'moment';
+import MockDate from 'mockdate';
 import { QuestionType } from '@tupaia/types';
 import { getBrowserTimeZone } from '@tupaia/utils';
 import { processSurveyResponse } from '../../utils';
 
-// Mock out moment so that that toISOString returns a consistent value for our tests
-jest.mock('moment', () => {
-  const mMoment = {
-    toISOString: jest.fn(() => 'theISOString'),
-  };
-  return jest.fn(() => mMoment);
-});
```

Comment:
Switched out moment as:
a) It's being deprecated and we want to stop using it
b) It was breaking the tests